src/net_processing.cpp

```diff
@@ -6061,7 +6186,9 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
                 entry.second.GetHash().ToString(), entry.first);
         }
         for (const GenTxid& gtxid : requestable) {
-            if (!AlreadyHaveTx(gtxid, /*include_reconsiderable=*/true)) {
+            // Skip m_recent_rejects_reconsiderable because we may be requesting a missing parent
```

Comment:
I have to admit I'm struggling on these "skip" comments and cases. Every time I read this I have trouble re-deriving the logic.

Reply:
think you forgot to push?

Reply:
> think you forgot to push?

Github published everything instead of adding my comment to the group -_- yes, pushing in a second.